sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/mailgun.yaml

```
@@ -12,7 +12,7 @@ provider:
       # Avoid matching values found on SourceGraph that look like dummy passwords or insertions like:
       # - key-00000000000000000000000000000000
       patternNot: 
-        "(?i)(\\w)\\1{6,}"
+        - "(?i)(\\w)\\1{6,}"
```

```suggestion
       # e.g. key-00000000000000000000000000000000
        - "(\\w)\\1{6,}"
```

Please remove the comment also